mcsema/Arch/X86/Semantics/String.cpp

```diff
@@ -859,4 +864,7 @@ void String_populateDispatchMap(DispatchMap &m) {
   m[llvm::X86::REPNE_CMPSW_64] = translate_CMPS16;
   m[llvm::X86::REPNE_CMPSD_64] = translate_CMPS32;
   m[llvm::X86::REPNE_CMPSQ_64] = translate_CMPS64;
+
+  m[llvm::X86::REP_PREFIX] = translate_REP_prefix;
```

Comment:
I mentioned this earlier in chat, but making `rep` an error would break things like `rep ret` or other superflous rep prefix use.